docs/src/reconciling-local-changes.md

```diff
@@ -0,0 +1,82 @@
+# Reconciling changes made outside of TPA
+
+In general, if a cluster was created using TPA it is advisable to manage
+configuration changes to that cluster using TPA. This page is designed
+to help users understand how TPA manages configuration, what the
+implications of making local changes to a TPA-managed cluster are, and
+how to reconcile such changes.
+
+## What constitutes a configuration change?
+*This definition seems questionable with changes which wouldn't normally
+appear in the config at all, like inserting some data into a table*
+Consider a cluster, 'A'. Now imagine a change is applied to it and call
+the resulting cluster 'B'. If the config.yml required to deploy a fresh
+cluster identical to 'A' is different to the config.yml required to
+deploy a cluster identical to 'B' then the change you made is a
+configuration change.
+
+## Are there configuration changes which cannot be performed using TPA?
+Yes, there are changes which would meet the above definition that cannot
+be implemented by TPA. The most notable class of such changes is
+*destructive changes*. In general TPA will not remove previously
+deployed elements of a cluster, even if these are removed from the
+config.yml. This sometimes surprises people because a strictly
+declarative system should always mutate the deployed artifacts to match
+the declaration. However, making destructive change to production
+database can have serious consequences so it is something we have chosen
+not to support.
+
+## What can happen if changes are not reconciled?
+A general issue with unreconciled changes is that if you deploy a new
+cluster using your existing config.yml, or provide your config.yml to
+EDB Support in order to reproduce a problem, it will not match the
+original cluster. In addition, there is potential for operational
+problems should you wish to use TPA to manage that cluster in future.
+
+The operational impact of unreconciled changes varies depending on the
+nature of the changes. In particular whether the change is destructive,
+and whether the change blocks TPA from.
```

Comment:
Again, it's tricky to generalize. This is my best attempt but I'm not sure if it's right enough of the time to be useful.